src/main/java/roomescape/member/MemberDao.java

```diff
@@ -40,6 +42,19 @@ public Member findByEmailAndPassword(String email, String password) {
         );
     }
 
+    public Member findByEmail(String email) {
+        return jdbcTemplate.queryForObject(
+                "SELECT id,name,email, role FROM member WHERE email = ?",
```

Comment:
```suggestion
                "SELECT id, name, email, role FROM member WHERE email = ?",
```

코드를 작성할 땐 최대한 일정한 코드 스타일을 유지해 봅시다!